installer/pkg/config-generator/ignition.go

```diff
@@ -126,18 +127,11 @@ func (c *ConfigGenerator) embedUserBlock(ignCfg *ignconfigtypes.Config) {
 func (c *ConfigGenerator) getTNCURL(role string, query string) string {
```

Comment:
No longer TNC `getMCSURL` ?

Reply:
I'm in favor of writing out anything project-specific.  Folks likely have seen URL before, which would make this `getMachineConfigServerURL`.

Reply:
I'd like to keep the rename out of scope here see https://github.com/openshift/installer/pull/242#issuecomment-422946285.

(I know I'm repeating myself here :-D)